README.md

```diff
@@ -45,7 +45,9 @@ To use the SAP AI SDK for Java, the following general prerequisites must be met:
     - [How to enable the AI Core service](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/initial-setup)
 - **SAP AI Core Credentials** to access the AI Core service.
     - [Connecting to SAP AI Core](#connecting-to-sap-ai-core)
-- **(Optional) Spring Boot** version 3 or higher if you are using Spring Boot.
+- **(Optional) Spring Boot** version 3 or higher.
+- **(Optional) SAP Cloud SDK** version `5.6.0` or higher.
```

Comment:
It's not really optional. What is optional is the user defining a version for it. But contrary to spring boot or cap, it has to be there. 

Personally, I like how cap does this here: https://cap.cloud.sap/docs/java/versions#minimum-versions

Reply:
"Optional" in the sense of custom version declaration, but sure I can phrase it differently.